### PR TITLE
Add option to mark a reminder as completed and update counter

### PR DESCRIPTION
**Pull request by StanislavPrime**
Add option to mark a reminder as completed and update both counter, but it makes the Complete counter number wrong, when the updated reminder is from the Reminders list

### CODE REVIEW

**Review by StanevPrime — APPROVED** [review]

File: examples/reminders/lib/base/common_ui_components/app_reminder_tile.dart [hunk 1]
```diff
@@ -109,8 +109,20 @@ class _AppReminderTileState extends State<AppReminderTile> {
...
+          SlidableAction(
+            onPressed: (context) =>
+              context.read<ReminderManageBlocType>().events.update(widget
+                  .reminder
+                  .copyWith(complete: !widget.reminder.complete))
+            ,
+            backgroundColor: context.designSystem.colors.activeButtonColor,
+            foregroundColor: context.designSystem.colors.canvasColor,
+            icon: Icons.check_box_outlined,
+            label: context.l10n.complete,
```
Comment (StanevPrime) [inline examples/reminders/lib/base/common_ui_components/app_reminder_tile.dart:124]: As we discussed the title should change based on the `complete` status

File: examples/reminders/lib/base/common_ui_components/app_reminder_tile.dart [hunk 2]
```diff
@@ -109,8 +109,20 @@ class _AppReminderTileState extends State<AppReminderTile> {
...
+        extentRatio: 0.7,
         motion: const ScrollMotion(),
         children: [
+          SlidableAction(
+            onPressed: (context) =>
+              context.read<ReminderManageBlocType>().events.update(widget
+                  .reminder
+                  .copyWith(complete: !widget.reminder.complete))
+            ,
+            backgroundColor: context.designSystem.colors.activeButtonColor,
```
Comment (StanevPrime) [inline examples/reminders/lib/base/common_ui_components/app_reminder_tile.dart:121]: As we discussed the color should change based on the `complete` status

**Review by StanevPrime — CHANGES_REQUESTED** [review]

File: examples/reminders/lib/base/common_ui_components/app_reminder_tile.dart (hunk 1 above)
Comment (StanislavPrime) [reply examples/reminders/lib/base/common_ui_components/app_reminder_tile.dart]: This was changed

File: examples/reminders/lib/base/common_ui_components/app_reminder_tile.dart (hunk 2 above)
Comment (StanislavPrime) [reply examples/reminders/lib/base/common_ui_components/app_reminder_tile.dart]: This was changed

File: examples/reminders/lib/base/data_sources/local/reminders_local_data_source.dart [hunk 3]
```diff
@@ -57,6 +58,7 @@ class RemindersLocalDataSource {
       id: id,
       title: title,
       complete: complete,
+      completeUpdated: completeUpdated,
```
Comment (StanevPrime) [inline examples/reminders/lib/base/data_sources/local/reminders_local_data_source.dart:61]: As we discussed  the rx_bloc_list now supports the needed infrastructure for checking old/new objects so please adapt this functionality accordingly

**Review by StanevPrime — CHANGES_REQUESTED** [review]

**Pull request closed without merging**